project_tag_security/models/project_project.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+from odoo import fields, models
+
+
+class Projectproject(models.Model):
+    _inherit = "project.project"
+
+    tag_ids = fields.Many2many(
+        domain="""['|', ('allowed_project_ids', 'in', [id]),
+        ('allowed_project_ids', '=', False)]"""
```

Comment:
Could you please try this to simplify the domain and avoid an `OR`?
```suggestion
        domain="""[('allowed_project_ids', 'in', [id] + [False])]"""
```

Reply:
I just tried it but it doesn't work (a pity), if you find something similar in core please indicate it for reference. It seems that only something similar works in the rules.

Reply:
I imagine this is evaluated as JS, so I'm not sure if using `[0]` or `[false]` will work.

Anyway, let's go!